apps/spruce/src/pages/waterfall/InactiveVersionsButton/index.tsx

```diff
@@ -63,3 +75,8 @@ const InactiveVersionLine = styled.div<{ containerHeight: number }>`
 const StyledVersionLabel = styled(VersionLabel)`
   padding-top: ${size.xs};
 `;
+
+const StyledBadge = styled(Badge)`
+  top: -${size.xs};
```

Comment:
[nit] I know the designs have this slightly above the rest of the elements but I think positioning all the filters will be easier if it's inline with the top of VersionLabel with some padding between it and the button